index.html

```diff
@@ -2,18 +2,86 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
```

Comment:
Missing a viewport meta tag. It's easy to forget because you often won't notice a difference until you view the site on a mobile device, but it's very important!